ydb/src/credentials.rs

```diff
@@ -12,19 +12,116 @@ use http::Uri;
 use jsonwebtoken::{encode, Algorithm, EncodingKey, Header};
 use secrecy::{ExposeSecret, SecretString};
 use serde::{Deserialize, Serialize};
+use std::env;
 use std::fmt::Debug;
 use std::ops::Add;
 use std::process::Command;
 use std::sync::{Arc, Mutex};
 use std::time::{Duration, Instant, SystemTime, UNIX_EPOCH};
-use tracing::{trace,debug};
+use tracing::{debug, trace};
+
+const YDB_ANONYMOUS_CREDENTIALS: &str = "YDB_ANONYMOUS_CREDENTIALS";
+const YDB_SERVICE_ACCOUNT_KEY_FILE_CREDENTIALS: &str = "YDB_SERVICE_ACCOUNT_KEY_FILE_CREDENTIALS";
+const YDB_METADATA_CREDENTIALS: &str = "YDB_METADATA_CREDENTIALS";
+const YDB_ACCESS_TOKEN_CREDENTIALS: &str = "YDB_ACCESS_TOKEN_CREDENTIALS";
+
+pub const YDB_METADATA_URL: &str =
```

Comment:
It isn't YDB metadata url, it is Yandex Cloud metadata URL

Reply:
This name is written in the [documentation](https://ydb.tech/docs/en/reference/ydb-sdk/auth)